### ISSUE TITLE
registry pod probes failing with operator version v1.2.0

### ISSUE THREAD

**Issue by tsusadivyago** [goharbor/harbor-operator#917]
**Reproduce:**

Below are the deployment files used
`
kubectl apply -f https://raw.githubusercontent.com/goharbor/harbor-operator/v1.2.0/manifests/cluster/deployment.yaml

kubectl apply -f https://raw.githubusercontent.com/goharbor/harbor-operator/v1.2.0/manifests/samples/full_stack.yaml
`
The domain is updated in full_stack.yaml.

**Issue:**

- Unable to log in to harbor with default credentials.
-  In the registry pod probe is failing which crashes and restarts later.

`NAME                                                    READY   STATUS             RESTARTS   AGE`
`harborcluster-harbor-harbor-registry-5484d94f86-p2r55   1/2     CrashLoopBackOff   393        20h`

![image](https://user-images.githubusercontent.com/58212484/174984687-25e98f81-3922-4aaa-b75d-b56dad590395.png)

![image](https://user-images.githubusercontent.com/58212484/174985337-ce627d21-c73d-48c6-b56e-076ab35bc4db.png)

![image](https://user-images.githubusercontent.com/58212484/174985561-6d9dd256-a17d-4c12-bbc7-cb11fc090804.png)

**Comment by fawadasaurus** [comment]
I am having the same issue.

**Comment by fawadasaurus** [comment]
Here is how I fixed it.

First, make sure you have a default ingress class set:
https://github.com/goharbor/harbor-operator/issues/913

Second, make sure your cert is properly configured.  I did this manually:
https://github.com/goharbor/harbor-operator/issues/923

After I did these two things, the pods came up successfully and everything was working as expected.

I am now on 2.5.1 and 1.3.0rc3 but not going to touch it since its working.
